Roys/app/src/main/java/com/dwg_karrier/roys/ListActivity.java

```diff
@@ -0,0 +1,86 @@
+package com.dwg_karrier.roys;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+import android.widget.AdapterView;
+import android.widget.ListView;
+
+import java.util.ArrayList;
+import java.util.Date;
+
+/**
+ * Created by userpc on 2017-01-16.
+ */
+
+public class ListActivity extends AppCompatActivity{
+  ListView lv;
+  ArrayList<item> data;
+  Date fin_time; /* expected finish time */
+  Date cur_time; /* current time*/
```

Comment:
nit: time */, please check https://google.github.io/styleguide/javaguide.html#s4.8.6-comments
For variable name, use https://google.github.io/styleguide/javaguide.html#s5.2.7-local-variable-names